### DISCUSSION TITLE
@tarojs/plugin-html安装后不能使用html标签

### DISCUSSION THREAD

**Discussion by LIANG-FU** [NervJS/taro#10651] [Q & A]
安装了@tarojs/plugin-html并在config/index.js里面加了下面的代码：
`plugins: [
    '@tarojs/plugin-html'
  ],`
但是在使用如下代码：
`import styles from './index.module.scss';
import '@tarojs/taro/html.css'
  
const Index = () => {
  return (
    <div className={styles['container']}>
      hello world!
    </div>
  )
}

export default Index;
`的时候小程序有警告，并不显示内容
![image](https://user-images.githubusercontent.com/39352965/141508067-5a252574-17db-40bd-8c38-2d2d3ae1a32c.png)

**Comment by Chen-jj** [comment]
@LIANG-FU 这类问题反馈提一下 issue 咯，并提供一下 demo。
